training/main_training_script.py

```diff
@@ -213,8 +215,8 @@ def main():
     # default_out_features = trainingModel.fc.out_features
     # trainingModel.fc = nn.Sequential(trainingModel.fc, nn.ReLU(), nn.Linear(in_features=default_out_features, out_features=1, bias=True))
     trainingModel = trainingModel.to(device)
-    optimizer = torch.optim.Adam(trainingModel.parameters(),lr=hyperparams['optimizer_lr']) 
-    scheduler = lr_scheduler.ReduceLROnPlateau(optimizer,mode='min',factor=0.5, patience=10,min_lr=0.001, verbose=True) #Start at large learning rate to quickly learn and only reduce when loss plateaus
+    optimizer = torch.optim.AdamW(trainingModel.parameters(),lr=hyperparams['optimizer_lr']) 
+    scheduler = lr_scheduler.ReduceLROnPlateau(optimizer,mode='min',factor=0.1, patience=10,min_lr=0.0002, verbose=True) #Start at large learning rate to quickly learn and only reduce when loss plateaus
```

Comment:
shouldn't we put these in the hyperparams.yml?

Reply:
o you are right I will do that in another PR at some point